testing/test_webinterface.py

```diff
@@ -32,14 +32,16 @@ def test_webinterface_start_stop():
     assert "Using template path" in app_log
     assert "Loading configuration from etc/testing/config-web.toml" in app_log
     # assert "Connecting to database sqlite:///secpi-database-testing.sqlite" in app_log
-    assert "Connecting to database mysql+pymysql://secpi:secret@localhost/secpi-testdrive" in app_log
+    # assert "Connecting to database mysql+pymysql://secpi:secret@localhost/secpi-testdrive" in app_log
+    assert "Connecting to database postgresql://secpi:secret@localhost/secpi-testdrive" in app_log
```

Comment:
We will need to get this from the environment somehow, that we will not have to change the log string each time.

Reply:
Something like here:
https://github.com/isarengineering/SecPi/blob/773758fbe38cac385ba14b0eaa4138435fbff60f/secpi/util/database.py#L89

Reply:
We will need that URI there.